service/operator/operator.go

```diff
@@ -8,51 +8,48 @@ import (
 	"time"
 
 	"github.com/cenk/backoff"
-	"github.com/giantswarm/kvmtpr"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/operatorkit/framework"
 	"github.com/giantswarm/operatorkit/informer"
 	"github.com/giantswarm/operatorkit/tpr"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/kubernetes"
 )
 
 // Config represents the configuration used to create a new service.
 type Config struct {
 	// Dependencies.
-	BackOff           backoff.BackOff
-	Informer          *informer.Informer
-	K8sClient         kubernetes.Interface
-	Logger            micrologger.Logger
-	OperatorFramework *framework.Framework
+	BackOff   backoff.BackOff
+	Framework *framework.Framework
+	Informer  *informer.Informer
+	Logger    micrologger.Logger
+	TPR       *tpr.TPR
```

Comment:
We inject the TPR now instead of creating it internally. This also removes the weird k8s client dependency which was always confusing because it was only used to create the TPR at runtime.